tilecloud_chain/views/admin.py

```diff
@@ -52,10 +54,14 @@ def run(self):
         try:
             display_command = ' '.join([shlex.quote(arg) for arg in self.command])
             LOG.info("Run the command `{}`".format(display_command))
+            env = {}
+            env.update(os.environ)
+            env['FRONTEND'] == 'noninteractive'
```

Comment:
I'm quite sure a `==` is **not** what you want

Reply:
Arg..., thanks :-)